src/main/java/data/SubjectRawDataBuilder.java

```diff
@@ -0,0 +1,80 @@
+package data;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Date;
+import java.util.Scanner;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
+/**
+ * Created by Timotej on 21-Oct-15.
+ */
+public class SubjectRawDataBuilder {
+
+    private Integer id;
+    private File closeMic;
+    private File BPMData;
+
+    public SubjectRawDataBuilder withId(int id) {
+        this.id = id;
+        return this;
+    }
+
+    public SubjectRawDataBuilder withCloseMic(File closeMic) {
+        this.closeMic = closeMic;
+        return this;
+    }
+
+    public SubjectRawDataBuilder withBPMData(File BPMData) {
+        this.BPMData = BPMData;
+        return this;
+    }
+
+    /**
+     * This methods finds closeMic file and BPMData files from the subject folder
+     * @param subjectDir
+     * @return
+     */
+    public SubjectRawDataBuilder asBCMDir(File subjectDir) {
+        for(File f : subjectDir.listFiles()) {
+            System.out.println(f.getName());
+            if(f.getName().equals("BPM.txt")) {
+                BPMData = f;
+            }
+            else if(f.getName().endsWith("close.wav")) {
+                closeMic = f;
+            }
+        }
+
+        return this;
+    }
```

Comment:
I wouldn't print anything to the console on master branch and would use a logger (if needed) instead. It also looks like this is for debugging. Also what happens if there are multiple files from this directory that match the if statement? If other files don't matter I would add a break.

Code style stuff: I would change the name from 'asBCMDir' to something like 'fromBCMDir' and close curly brace before else keyword.